.gitignore

```diff
@@ -17,6 +17,7 @@ tools/vendor/
 # Mbin specific
 .env
 /public/media/*
+/public/media
```

Comment:
I think you could also do a single rule of `/public/media*` to cover both situations but that is a minor nitpick

Reply:
but that could potentially also ignore media-blabla files in the public folder right?

Reply:
or media.css .. just an example.